docs/pages/admin-guides/access-controls/rbac/resource-rbac.mdx

```diff
@@ -0,0 +1,108 @@
+
+---
+title: Getting Started With Per-Resource Role-Based Access Control
+description: Implementing fine-grained access control with RBAC.
+---
+
+
```

Comment:
This guide needs an introductory paragraph to explain its scope/why a reader would want to continue with this guide.

Reply:
Added!